### ISSUE TITLE
HC theme viewlet focus outline doesn't work on first item

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode#8419]
#8215
- VSCode Version: Code - Insiders 1.3.0-insider (d0c2b891903d73e83dcdc7786398ebfa764fc4ee, 2016-06-28T05:01:27.563Z)
- OS Version: Linux x64 4.4.0-24-generic, Ubuntu 16.04

Correct: 

![image](https://cloud.githubusercontent.com/assets/2193314/16427795/e73659e0-3d23-11e6-97d3-0a8343b91f67.png)

Top missing on first item:

![image](https://cloud.githubusercontent.com/assets/2193314/16427798/f149c534-3d23-11e6-936a-4c2e9ddae386.png)

**Comment by bpasero** [comment]
@Tyriar not happening anymore, is it?
